### PR TITLE
Emergency Shuttle Fuckery Panel

### PR DESCRIPTION
**Pull request by DeityLink**
image
For @N3X15 

Might want to compile it on the test server first. I had to reorganize/clean the emergency shuttle controller a bit. Hopefully it won't break anything.

When moving the emergency shuttle, you can choose to "just" move it, labeled in the logs with "no round triggers" which means there won't be announcements, the ETA won't appear, it won't end the round, etc...
Alternatively, you can now force a round end by sending the shuttle to centcom. Even if it's already there. 

It's 5am and I have a splitting headache. I'm off.

:cl:
* experiment: Admins now have a panel that allows them to easily control the Emergency Shuttle, its docking ports, as well as the Escape Pods.

### CODE REVIEW

**Comment by DeityLink** [comment]
- Made it so that the voting assets preload even if the shuttle hasn't been through transit. And so they only preload once.

I'll try adding those sounds @jknpj gave me

**Comment by DeityLink** [comment]
there, good to go. people will now hear the shuttle's engines warm up 5 seconds before it takes off (can be heard from up to 50 tiles away from the docking port, the volume linearly increases as you get closer). Should give a bit of an edge to those funny people waiting for the last second before boarding up.

**Comment by ghost** [comment]
Put the sounds in their own PR

**Comment by DeityLink** [comment]
sure

**Comment by N3X15** [comment]
Looks good to me, now waiting for the thread to re-evaluate and determine whether it torques their boxers still or not.

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1614,6 +1614,12 @@ Game Mode config tags:
 	return (team in sent_strike_teams)
 
 
+/proc/area_in_map(var/area/A)
+	for (var/turf/T in A.area_turfs)
```
Comment (PJB3005) [inline code/__HELPERS/unsorted.dm:1618]: Erm... Can't you just check `area.x` to get whether it's on the map?

**Pull request merged**